metaTag.php

```diff
@@ -0,0 +1,2 @@
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<meta name="google-site-verification" content="J8diIFiI9tPzvGFlnEAm8OwhFSpN0vjBpymuKBwCSbU" />
```

Comment:
@aniarya82 

Rather than this, add the google HTML page.

https://github.com/students-alumni-cell-iitkgp/aam-2016/pull/34#commitcomment-12484212